lib/coney/consumer.ex

```diff
@@ -1,9 +1,11 @@
 defmodule Coney.Consumer do
+  @type stacktrace :: []
+  @type error_tuple :: {exception :: struct(), stacktrace}
```

Comment:
let's add name for stacktrace :) `{exception :: struct(), stacktrace :: stacktrace()}`